<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisSettings</class>
 <widget class="QWidget" name="AxisSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="axisSelection"/>
   </item>
   <item>
    <widget class="QGroupBox" name="axisBox">
     <property name="title">
      <string>Axis Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="axisVisibility">
        <property name="text">
         <string>Show Axis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="majorTicks">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show Major Ticks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="minorTicks">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show Minor Ticks</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
